apps/web/layouts/organization/standalone/components/MobileMenu.tsx

```
@@ -0,0 +1,98 @@
+                    paddingTop={1}
+                    paddingBottom={1}
+                  >
+                    {`${homeLabelPrefix}${homeLabel ? ': ' + homeLabel : ''}`}
+                  </Text>
+                </Box>
+              </LinkV2>
+              <Divider />
+              {links.map((link) => (
+                <>
+                  <Box className={styles.links}>
+                    <LinkV2 key={link.label} href={link.href}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)